modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -2359,7 +2458,8 @@ var UIDynamicList = UIElement.extend(/** @lends LuCI.ui.DynamicList.prototype */
 			return;
 
 		if (item) {
-			this.removeItem(dl, item);
+ 			if (item.getBoundingClientRect().right - ev.clientX<25)
+				this.removeItem(dl, item);
 		}
```

Comment:
This may need firming up. This is to ensure we don't start to drag but delete the item. Maybe we can derive the general dimensions of the X part, since it may vary through different themes. Try this:
```js

		if (item) {
			// Get bounding rectangle of the item
			var rect = item.getBoundingClientRect();

			// Get computed styles for the ::after pseudo-element
			var afterStyles = window.getComputedStyle(item, '::after');
			var afterWidth = parseFloat(afterStyles.width) || 0;

			// Check if the click is within the ::after region
			if (rect.right - ev.clientX <= afterWidth) {
				this.removeItem(dl, item);
			}
		}
```

I get the move cursor anyway juuust at the right edge in bootstrap, but that's a byproduct of how the design lays elements out. So that's OK.